openedx/core/djangoapps/content_tagging/rest_api/v1/views.py

```diff
@@ -130,8 +139,57 @@ def orgs(self, request, **_kwargs) -> Response:
 class ObjectTagOrgView(ObjectTagView):
     """
     View to create and retrieve ObjectTags for a provided Object ID (object_id).
-    This view extends the ObjectTagView to add Organization filters for the results.
+    This view extends the ObjectTagView to add Organization filters for the results and
+    new actions like: export.
 
     Refer to ObjectTagView docstring for usage details.
     """
     filter_backends = [ObjectTagTaxonomyOrgFilterBackend]
+
+    def get_queryset(self):
+        if self.action == "retrieve":
+            return super().get_queryset()
+
+        # For other actions, return a dummy queryset only for permission checking
+        dummy_queryset = QuerySet(model=ObjectTag)
+
+        return dummy_queryset
```

Comment:
In the view from oel_tagging, we are returning a query set with a list of `ObjecTags` in this method. The problem is that every other function/action with `detail=True` will call `get_queryset`, running unnecessary code.

I'm unsure if we should fix it here, in oel_tagging, or maybe refactor it to stop using a ModelView since we are not following the default "model" GET/UPDATE/DELETE here.

This code solve the problem for the export action.

Reply:
Done: 04ca0724162afb12891984f26fcad4393297eae4